### ISSUE TITLE
Land switching breaks Godot Input

### ISSUE THREAD

**Issue by olehpatrushev** [godotengine/godot#49771]
### Godot version

3.3.stable

### System information

Ubuntu 20.04

### Issue description

According to title, when I change system lang to some others from English (in my case Russian) on Ubuntu, I find that Godot shortcuts not working in editor (even Ctrl+V), and runtime Input system also brakes (event W not working).

### Steps to reproduce

Change system language to some other (maybe cyrillic on Ubuntu) and try to manipulate with Godot.

**Comment by olehpatrushev** [comment]
> Can you reproduce this if you exit and restart Godot with the keyboard layout in question?
> 
> Godot does not use physical input keys, which means you need to press the key for your _current_ keyboard layout for shortcuts to work. Optional physical keys were added to #18020, but they're not used anywhere in the editor yet.

Yes. It is reproduced.

**Comment by Calinou** [comment]
@olehpatrushev After switching keyboard layouts, do you press <kbd>V</kbd> as it appears on your original Russian keyboard layout, or <kbd>V</kbd> as it appears on a QWERTY layout?

**Comment by olehpatrushev** [comment]
@Calinou yes I do the same steps (shortcuts) as in other programs, such as Google Chrome, and it doesn't work in Godot, but in Google Chrome works.

**Comment by Zireael07** [comment]
You didn't answer the question - do you press the V key on the keyboard, or some other key?

**Comment by olehpatrushev** [comment]
@Zireael07 there is no V on russian keyboard layout

**Comment by Zireael07** [comment]
Do you press the key on the keyboard that is where a V would be in QWERTY, or what? Because if there's no V in Russian layout, that begs the question of how do Ctrl+V work in other apps?

**Comment by olehpatrushev** [comment]
@Zireael07 feel free to type any characters on keyboard in non-English layout, they would not work in Godot, I assure you.
